### PR TITLE
remove obsolete mk2/mk3 code paths from firmware

### PR DESCRIPTION
**Pull request by scgbckbone**
Removed old Mk2/Mk3 code that is no longer used.

### CODE REVIEW

File: shared/seed.py [hunk 1]
```diff
@@ -637,18 +635,8 @@ def clear_seed():
...
-
-        if version.has_608:
-            # wipe the long secret too
-            nv = bytes(AE_LONG_SECRET_LEN)
-            pa.ls_change(nv)
+    callgate.fast_wipe(True)
+    # NOT REACHED
 
     dis.busy_bar(False)
     dis.fullscreen('Reboot...')
```
Comment (doc-hex) [inline shared/seed.py:642]: these 2 lines are unreachable

**Comment by doc-hex** [comment]
- `mk-sigheader.py` and related files are a bit messed up? Hard to tell from the diff. 
- all of the changes to `/stm32/` are not helpful since they do not reduce memory size.

File: shared/seed.py (hunk 1 above)
Comment (scgbckbone) [reply shared/seed.py:642]: fixed in https://github.com/Coldcard/firmware/pull/263

**Comment by scgbckbone** [comment]
> * `mk-sigheader.py` and related files are a bit messed up? Hard to tell from the diff.
> 
>     * all of the changes to `/stm32/` are not helpful since they do not reduce memory size.

reverted whole `stm32` (we can do this in solo PR later)

**Comment by scgbckbone** [comment]
https://github.com/Coldcard/firmware/pull/263

**Pull request closed without merging**